### PR TITLE
{data,tools}[foss/2022b] pyodbc v4.0.39, unixODBC v2.3.11

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/u/unixODBC/unixODBC-2.3.11-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+homepage = "https://www.unixodbc.org"
+description = """unixODBC provides a uniform interface between
+application and database driver"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['https://www.unixodbc.org/']
+checksums = ['d9e55c8e7118347e3c66c87338856dad1516b490fb7c756c1562a2c267c73b5c']
+
```
Comment (verdurin) [inline easybuild/easyconfigs/u/unixODBC/unixODBC-2.3.11-foss-2022b.eb:20]: @PetrKralCZ please note the bot's complaints about missing sanity check paths.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4007.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/7b357435f21214c2105af2f7da4ba3c0 for a full test report.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/fe9d72c2f92a4760b7fbc1281ba50788 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**